src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -368,7 +388,7 @@ public Transaction build() {
 
       Operation[] operations = new Operation[mOperations.size()];
       operations = mOperations.toArray(operations);
-      Transaction transaction = new Transaction(mSourceAccount.getKeypair(), operations.length * BASE_FEE, mSourceAccount.getIncrementedSequenceNumber(), operations, mMemo, mTimeBounds);
+      Transaction transaction = new Transaction(mSourceAccount.getKeypair(), operations.length * baseFee, mSourceAccount.getIncrementedSequenceNumber(), operations, mMemo, mTimeBounds);
```

Comment:
It'd be fantastic if it printed a deprecation warning (like was [added](https://github.com/stellar/js-stellar-base/pull/167) to the JS SDK) when `baseFee` hasn't been set explicitly

Reply:
will do.